# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ghostpcl/prefix.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ghostpcl_1.40/Makefile.vanilla	2004-03-01 23:12:42.000000000 +0000
+++ ghostpcl_1.40/Makefile	2007-04-15 17:45:22.000000000 +0000
@@ -1,3 +1,5 @@
+prefix = /usr/local
+
 product:
 	make -C main -f pcl6_gcc.mak	# build PCL and PCLXL. 
 
@@ -13,7 +15,7 @@
 	make -C main -f pcl6_gcc.mak pg-fp
 
 install:
-	install main/obj/pcl6 /usr/local/bin
+	install main/obj/pcl6 $(prefix)/bin
 
 test: 
 	cd tools; ../main/obj/pcl6 -dTextAlphaBits=4 owl.pcl tiger.px3 # test with PCL and PXL test file 
